### DISCUSSION TITLE
🚀 Feature Request: Ability to customize the Post Comments widget and its full compatibility with WooCommerce (Migrated from #16304)

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#23261] [Feature Request]
- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
hello
One of the biggest weaknesses of hello elementor site design is that the comments section is not customizable. And the only solution is coding.
Great if you develop this widget. In such a way that all its parts (comments section) can be customized.
In the meantime, the most important thing is that this widget is not fully compatible with WooCommerce. The full compatibility of this widget with WooCommerce will be excellent.

Items required in this personalization include :

1-Ability to customize the avatar section (user image).
2- Ability to fully personalize the admin comment section.
3- Full personalization of comment fields (name, email, website) and user message section.
4- Ability to delete or add fields. For example, we can determine what information to get from the user so that he can post his opinion. (Name, website, email) or none of these.
5- **Star rating**
6- Full star rating customization.
7- Full compatibility with WooCommerce.
8- **Compatibility with WooCommerce Buyer Approved Owner Section.**
9- **When the user comments, they can edit or delete their message.**
10- **Users can reply to other users (reply)**
11- Ability to customize the comments time section.
12- Ability to send image by users.
13- Ability to display a summary of user comments (For example, x people gave 5 stars, x people gave 3 stars or ...)  Like the jet review plugin.
14- Ability to deactivate the user avatar section.

The above may be a little extravagant. But they will definitely be needed. I hope we can have at least some simple customization soon.
If the goal is to build many websites with elementor and especially hello eleemtor, then these are definitely needed. Because they are not customizable at the moment.

thank you

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #16304 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by hadiomid** [comment]
+1
